docs/reference-architectures/virtual-machines-linux/multi-vm.md

```diff
@@ -137,29 +124,59 @@ For incoming Internet traffic, the load balancer rules define which traffic can
 
 ## Deploy the solution
 
-A deployment for this architecture is available on [GitHub][github-folder]. It includes a VNet, NSG, load balancer, and two VMs. It can be deployed with either Windows or Linux VMs. To deploy the architecture, follow these steps: 
+A deployment for this architecture is available on [GitHub][github-folder]. It includes a VNet, NSG, and three VMs in a scaleset behind a load balancer,described below:
+
+  * **VNet**. A sample virtual network with a sngle subnet named **web** used to host the VMs.
+  * **NSG**. A saple NSG with an incoming rules to allow HTTP traffic to the VM scaleset.
+  * **VMs**. Sample Linux VMs running the latest version of Windows Server 2016 Datacenter Edition in a scaleset with autoscale turned on, behind a load balancer.
+
+### Prerequisties
+
+Before you can deploy the reference architecture to your own subscription, you must perform the following steps.
+
+1. Clone, fork, or download the zip file for the [AzureCAT reference architectures][ref-arch-repo] GitHub repository.
 
-1. Click the button below:<br><a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmspnp%2Freference-architectures%2Fmaster%2Fvirtual-machines%2Fmulti-vm%2Fazuredeploy.json" target="_blank"><img src="http://azuredeploy.net/deploybutton.png"/></a>
-2. Once the link has opened in the Azure portal, you must enter values for some of the settings: 
-   * The **Resource group** name is already defined in the parameter file, so select **Create new** and enter `ra-multi-vm-rg` in the text box.
-   * Select the region from the **Location** drop down box.
-   * Do not edit the **Template Root Uri** or the **Parameter Root Uri** text boxes.
-   * Select either **windows** or **linux** in the **Os Type** drop down box. 
-   * Review the terms and conditions, then click the **I agree to the terms and conditions stated above** checkbox.
-   * Click the **Purchase** button.
-3. Wait for the deployment to complete.
-4. The parameter files include a hard-coded administrator user name and password, and it is strongly recommended that you immediately change both. Click the VM named `ra-multi-vm1` in the Azure portal. Then, click **Reset password** in the **Support + troubleshooting** blade. Select **Reset password** in the **Mode** dropdown box, then select a new **User name** and **Password**. Click the **Update** button to save the new user name and password. Repeat for the VM named `ra-multi-vm2`.
+2. Make sure you have the Azure CLI 2.0 installed on your computer. To install the CLI, follow the instructions in [Install Azure CLI 2.0][azure-cli-2].
 
+3. Install the [Azure building blocks v2][azbb] npm package.
```

Comment:
Aren't we calling this "Azure Building Blocks" (no v2)?

Reply:
yeah